.github/workflows/ci.yml

```diff
@@ -37,17 +37,26 @@ jobs:
   lint-pydocstyle:
     name: lint (pydocstyle)
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 1
-      - uses: actions/setup-python@v2
+      - uses: conda-incubator/setup-miniconda@v2
         with:
-          python-version: "3.7.x"
-          architecture: "x64"
-      - run: pip install pytest pytest-cov pydocstyle numpydoc==1.1.0
-      - run: pytest scripts/test
-      - run: pip install -e .[all]
```

Comment:
We need PyArrow to be installed to import and do a doc-style check for files from `backends/pyarrow`, the fresh install of Modin does not install PyArrow (#2090), so we must install it manually for this job. PyArrow from pip does not provide `pyarrow.gandiva` which is also required by PyArrow backend, so we have to use conda here.

Reply:
Is it expensive to install PyArrow in this case? If this is too expensive we can create "fake" modules like it was done for `cuDF` ([link](https://github.com/modin-project/modin/pull/2993/files#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3R23-R34))

Reply:
Also if some module is needed for docs checking only, it can me mocked by `unittest.mock.Mock` as @anmyachev [suggested](https://github.com/modin-project/modin/pull/3005#issuecomment-826874568).

Reply:
Thanks, it helped!